modules/cli/pages/gateway.adoc

```diff
@@ -0,0 +1,352 @@
+----
+
+where:
+
+* <config-filename> - the name of the YAML file, including suffix, that you generated in the previous step.
+* <destination-directory> - the location where you want to save the resulting gateway bundle, for example `~/gateways`.
+
+For example:
+[source, bash]
+----
+$ skupper gateway generate-bundle database.yaml ./
+----
+
+This bundle contains the gateway definition YAML and a  certificate that allow access to the {service-network}.
+
+--
+
+. Copy the gateway definition file, for example, `mylaptop-jdoe.tar.gz` to the machine that hosts the service you want to expose on the {service-network}.
+
+. From the machine that hosts the service you want to expose:
++
+--
+[source, bash]
+----
+$ mkdir gateway
+
+$ tar -xvf <gateway-definition-file> --directory gateway
+$ cd gateway
+$ sh ./launch.py
+----
+
+NOTE: Use `./launch.py -t podman` or `./launch.py -t docker` to run the Skupper router in a container.
+
+Running the gateway bundle uses the gateway definition YAML and a certificate to access and expose the service on the {service-network}.
+
+--
+
+. Check the status of the gateway service:
++
+--
+
+To check a _service_ type gateway:
+[source, bash]
+----
+$ systemctl --user status <gateway-definition-name>
+----
+
+To check a _podman_ type gateway:
+[source, bash]
+----
+$ podman inspect
+----
+
+To check a _docker_ type gateway:
+[source, bash]
+----
+$ docker inspect
+----
+
+
+NOTE: You can later remove the gateway using `./remove.py`.
+
+--
+
+. From the machine with cluster access, check the status of Skupper gateways:
++
+--
+[subs=attributes+]
+----
+$ skupper gateway status
+Gateway Definition:
+╰─ machine-user type:service version:{service-version}
+   ╰─ Bindings:
+      ╰─ mydb:3306 tcp mydb:3306 localhost 3306
+----
+This shows that there is only one exposed service and that service is only exposing a single port (BIND). There are no ports forwarded to the local host.
+--
+
+NOTE: If you need to change the gateway definition, for example to change port, you need to remove the existing gateway and repeat this procedure from the start to redefine the gateway.
+
+// Type: procedure
+[id="gateway-reference"] 
+== Gateway YAML reference
+
+The xref:exporting-gateway[] describes how to create a gateway to apply on a separate machine using a gateway definition YAML file.
+
+The following are valid entries in a gateway definition YAML file.
+
+name:: Name of gateway
+bindings.name:: Name of binding for a single host.
+bindings.host:: Hostname of local service.
+bindings.service:: Definition of service you want to be available on service network.
+bindings.service.address:: Address on the service network, name and port.
+bindings.service.protocol:: Skupper protocol, `tcp`, `http` or `http2`.
+bindings.service.ports:: A single port that becomes available on the service network. 
+bindings.service.exposeIngress:: (optional) The ingress mode for the service.
```

Comment:
@ajssmith What does the exposeIngress mode do?
You can specify it in a gateway yaml file, but I'm not sure what values are valid?

Reply:
The gateway has (yet another) convention where ingress/egress conveyed direction of service connectivity with respect to application network as gateway is on the edge. TL/DR, ingress == listener, egress == connector wrt router configuration.